packages/sdk/src/bot/teamsBotSsoPrompt.ts

```diff
@@ -294,7 +294,8 @@ export class TeamsBotSsoPrompt extends Dialog {
       "Get Teams member account user principal name: " + account.userPrincipalName
     );
 
-    const signInResource = this.getSignInResource(account.userPrincipalName);
+    const loginHint: string = account.userPrincipalName ? account.userPrincipalName : "";
```

Comment:
Nit: `const loginHint: string = account.userPrincipalName ?? ""` may work and be simpler.

Reply:
This seems like a syntactic sugar and may not be as clear as the original one. Seems conditional operation is generic in many languages. I prefer to keep the original one. Thx.